spec/Flubit/Client/ApiClientSpec.php

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
In few places you've used name `mock` and `client` for initialised objects. But names has first letter uppercase :)